src/status_im2/contexts/onboarding/new_to_status/style.cljs

```diff
@@ -0,0 +1,43 @@
+(ns status-im2.contexts.onboarding.new-to-status.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(def full-screen {:flex 1 :position :relative})
+
+(def image-background
+  {:height "100%"
+   :width  "100%"})
+
+(def layer-background
+  {:position         :absolute
+   :top              0
+   :bottom           0
+   :left             0
+   :right            0
+   ;; TODO: something is wrong here:
+   :background-color "rgba(25,36,56,0.8)" ;colors/neutral-80-opa-80
+  })
+
+(def navigation-bar {:height 56})
+
+(def options-container
+  {:padding-top        12
+   :padding-horizontal 20})
+
+(def title
+  {:color         :white
```

Comment:
better use colors/white from `quo2.foundations.colors` for consistency

Reply:
Solved in 3bf47252da26ca9387b10fed795b4c22e4393e35